internal/awsnuke/client_integration_test.go

```diff
@@ -0,0 +1,89 @@
+package awsnuke_test
+
+import (
+	"testing"
+	"time"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/credentials"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/ec2"
+	"github.com/gruntwork-io/aws-nuke/internal/awsnuke"
+)
+
+var AWSAccountId = "353720269506"
+
+func TestDestroyEC2Instances(t *testing.T) {
+	createNewInstance(t, "us-east-1", "ami-6057e21a")
+
+	nuke := awsnuke.New(AWSAccountId)
+	instances, err := nuke.ListNonProtectedEC2Instances()
+	if err != nil {
+		t.Fatalf("unexpected error: %v", err)
+	}
+
+	err = nuke.DestroyEC2Instances(instances)
+	if err != nil {
+		t.Fatalf("unexpected error: %v", err)
+	}
+
+	instances, err = nuke.ListNonProtectedEC2Instances()
+	if err != nil {
+		t.Fatalf("unexpected error: %v", err)
+	}
+
+	if expect, actual := 0, len(instances); expect != actual {
```

Comment:
We use [testify](https://github.com/stretchr/testify) to turn these sorts of checks into one-line `assert.Equals(...)` calls.